src/client/game.cpp

```diff
@@ -3353,6 +3353,7 @@ bool Game::nodePlacement(const ItemDefinition &selected_def,
 		} else {
 			node = map.getNode(p, &is_valid_position);
 			if (is_valid_position && !nodedef->get(node).buildable_to) {
+				selected_def.sound_place_failed;
```

Comment:
You're taking a value and ... doing nothing with it.
The compiler will optimize this out and it will surely not fix the issue.

Reply:
```suggestion
				soundmaker->m_player_rightpunch_sound = selected_def.sound_place_failed;
```